### PR TITLE
Stickyfooter fix

### PR DESCRIPTION
**Pull request by Paul-Guerra**
Protip: automate the csslint job to check your css during dev so all the changes don't add up

### CODE REVIEW

File: webclient/css/sticky-footer.css [hunk 1]
```diff
@@ -1,20 +1,21 @@
-*/
```
Comment (Paul-Guerra) [inline webclient/css/sticky-footer.css]: This is the actual fix. You had a closing comment tag without an open one

File: package.json [hunk 2]
```diff
@@ -4,11 +4,15 @@
   "description": "A chat and team coordination app and server",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "css:lint": "csslint --format=compact ."
   },
```
Comment (Paul-Guerra) [inline package.json]: you can run this by typing
````
npm run css:lint
````

File: webclient/css/sticky-footer.css (hunk 1 above)
Comment (markfguerra) [inline webclient/css/sticky-footer.css]: D'oh! Good catch, thanks!

File: package.json [hunk 3]
```diff
@@ -4,11 +4,15 @@
...
   "author": "Mark Guerra",
   "dependencies": {
     "express": "^4.14.0",
     "socket.io": "^1.7.2"
+  },
+  "devDependencies": {
+    "csslint": "^1.0.5"
```
Comment (markfguerra) [inline package.json]: Great, css lint seems like it makes sense as a dev dependency

File: package.json (hunk 2 above)
Comment (markfguerra) [inline package.json]: Very convenient, thanks

**Pull request merged**